backend/api/helper/permissions.py

```diff
@@ -0,0 +1,16 @@
+from flask_security import current_user
+
+
+def current_user_has_permission_to_change(entry) -> bool:
```

Comment:
I dislike this very much compared to the is_permitted method before. The previous method call was even shorter due to the name and more versatile.